examples/guestbook/README.md

```diff
@@ -75,6 +82,8 @@ To start the redis master, use the file `examples/guestbook/redis-master-control
 
 Although we have a single instance of our redis master, we are using a [replication controller](../../docs/user-guide/replication-controller.md) to enforce that exactly one pod keeps running. E.g., if the node were to go down, the replication controller will ensure that the redis master gets restarted on a healthy node. (In our simplified example, this could result in data loss.)
 
+The file `examples/guestbook/redis-master-controller.yaml` defines the redis master replication controller:
```

Comment:
I think it will be more logical to talk about the concepts in the order of pod, replication controller and service.